index.html

```diff
@@ -61,123 +72,100 @@ <h2 class="container job-title">I am a Software developer </h2>
-            </div>
-        </div>
-    
-      <!-- Third-Profesional-Art-Printing-Data -->
-      <div class="card-section">
-        <div class="work-card">
-            <div class="card-content flex-container">
-                <h4 class="title">Profesional Art Printing Data</h4>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                <ul>
-                    <li class="html">html</li>
-                    <li class="Boot">Bootstrap</li>
-                    <li class="Ruby">Ruby</li>
+        <div class="recent-work" id="recent-works">
+            <a href="" class="women" ><img src="./image/Img_Placeholder_1.png" alt="Women doing yoga"></a>
+            <div>
+                <h4>Multi-Post Stories</h4>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="tech-used">
+                    <li>CSS</li>
+                    <li>html</li>
+                    <li>bootsrap</li>
+                    <li>Ruby</li>
                 </ul>
-            </div>
-                <button class="project-button" type="button" id="thirdTitle">See Project</button>
-            </div>
-        </div>
-
- <!-- Fourth-Profesional-Art-Printing-Data -->
- <div class="card-section">
-    <div class="work-card">
-        <div class="card-content flex-container">
-            <h4 class="title">Profesional Art Printing Data</h4>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-            <ul>
-                <li class="html">html</li>
-                <li class="Boot">Bootstrap</li>
-                <li class="Ruby">Ruby</li>
-            </ul>
-            </div>
-            <button class="project-button" type="button"  id="fourthTitle">See Project</button>
+                <button class="project-button " type="button">See Project</button></div>
         </div>
-    </div>
-
-<!-- Fifth-Profesional-Art-Printing-Data -->
-<div class="card-section">
-    <div class="work-card">
-        <div class="card-content flex-container">
-            <h4 class="title">Profesional Art Printing Data</h4>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-            <ul>
-                <li class="html">html</li>
-                <li class="Boot">Bootstrap</li>
-                <li class="Ruby">Ruby</li>
-            </ul>
-        </div>
-    
-        <button class="project-button" type="button" id="fifthTitle">See Project</button>
     
+        <div class="cards-section">
+            <div class="work-card">
```

Comment:
- [ ] Well done for designing the work card properly, and it can be improved by adding its hover effect to your work card.


![image](https://user-images.githubusercontent.com/34779928/193107259-bc00ef95-8fbe-459a-81b3-93aa413983fa.png)

Reply:
@aimalamiri Hello sir
hope you are well.
sir please see my repo, I brought some changes to make hover but still has some issue.
and my friends, coding partner and stand-up teem also didn't understand.
so please help me, what should I do??

https://github.com/ZuhRa-HashiMi/portfolio_mobile/pull/4

Reply:
Hi @ZuhRa-HashiMi, it looks like that you're almost there. Simply try to troubleshoot the hover effect. You may, for example, limit the card height and change the 'display' state of particular components to hide or show them.